setup_jenkins.sh

```diff
@@ -0,0 +1,174 @@
+#! /bin/bash
+
+# Colorized output colors
```

Comment:
I think this file can be completely replaced by just putting the configuration into the Dockerfile as explained in the README comment

Reply:
I agree, I've removed it.